examples/anonymous-functions/anonymous_functions.bal

```diff
@@ -0,0 +1,17 @@
+import ballerina/io;
+
+public function main() {
+    // Define the anonymous function.
+    function (string, string) returns (string) anonFunction =
+            function (string x, string y) returns (string) {
+                return x + y;
+            };
+    io:println("Output: " + anonFunction("Hello ", "World.!!!"));
+
+    // If the anonymous function contains only the return statement in the body,
+    // the arrow function expression can be used instead of the anonymous function.
+    // The types of the input parameters will be inferred from the lhs.
```

Comment:
Minor change:
"// If an anonymous function contains only the return statement in the body,
 // you can use the arrow function expression instead of the anonymous function.
 // You can infer the type of input parameters from the lhs."